extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -119,6 +125,40 @@
 
     }
 
+    @ConfigGroup
+    public static class Logout {
+
+        /**
+         * The logout endpoint at the application. If provided, the application is able to initiate the logout through this
+         * endpoint in conformance with the OpenID Connect RP-Initiated Logout specification.
+         */
+        @ConfigItem
+        Optional<String> path;
+
+        /**
+         * The {@code URI} to redirect the user after the logging out from the OpenID Connect Provider. This {@code URI} must be
+         * properly registered at the OpenID Connect Provider as a valid redirect URI.
+         */
+        @ConfigItem
+        Optional<String> redirectUri;
```

Comment:
It is not used as a `URI` in the code. I also propose to rename the property to something less technical such as  `landingPath` or `postLogoutPath` or similar and doc it as `Relative path of the application endpoint where the user should be redirected to after the logging out from the OpenID Connect Provider` or similar

Reply:
Makes sense.